goldmane/pkg/aggregator/index_test.go

```diff
@@ -160,11 +185,404 @@ func TestIndexAddRemove(t *testing.T) {
 	idx.Remove(trickyFlow)
 
 	// Verify that the index is empty.
-	flows = idx.List(IndexFindOpts{})
+	flows, _ = idx.List(IndexFindOpts{})
 	Expect(flows).To(HaveLen(0))
 
 	// Remove flows we know aren't in the index, to make sure we're idempotent.
 	for _, flow := range allFlows {
 		idx.Remove(flow)
 	}
 }
+
+func TestIndexPagination_General(t *testing.T) {
+	defer setupTest(t)()
```

Comment:
We should call this within the `t.Run` - it's meant to be called per-test, not per suite.

This can cause problems if we eventually add other per-test state to this function.